tools/ibc-test/README.md

```diff
@@ -33,7 +33,7 @@ cd $IBC_CONTRACTS_SRC_PATH && yarn install && yarn compile
 # Axon
 export AXON_SRC_PATH=<axon source path>
 cd $AXON_SRC_PATH
-git checkout forcerelay-test
+git checkout forcerelay-dev
```

Comment:
Why did you close this PR?
I think it's necessary to update the Axon branch to `forcerelay-dev`.